src/components/DecisioningEngine/index.js

```diff
@@ -58,7 +58,7 @@ const createDecisioningEngine = ({ config, createNamespacedStorage }) => {
         run: decisionContext =>
           applyResponse({
             propositions: decisionProvider.evaluate(
-              contextProvider.getContext(decisionContext)
+              contextProvider.getContext(flattenObject(decisionContext))
```

Comment:
This should be written

```js
applyResponse({
  propositions: decisionProvider.evaluate(
    flattenObject(contextProvider.getContext(decisionContext))
  ),
});
```

otherwise you don't get all the global context flattened.

please write a test that tests the `renderDecisions` command to verify.